source/shared/cpp/ObjectModel/BaseActionElement.cpp

```diff
@@ -95,7 +95,12 @@ void BaseActionElement::PopulateKnownPropertiesSet()
          AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::IconUrl)});
 }
 
-void BaseActionElement::GetResourceInformation(std::vector<RemoteResourceInformation>&)
+void BaseActionElement::GetResourceInformation(std::vector<RemoteResourceInformation>& resourceInfo)
 {
-    return;
+    if (!m_iconUrl.empty()) {
```

Comment:
the `{` should be on the next line :)

Reply:
you're right!

thanks

---
In reply to: [211026141](https://github.com/Microsoft/AdaptiveCards/pull/1812#discussion_r211026141) [](ancestors = 211026141)